Podfile

```diff
@@ -1,5 +1,6 @@
 platform :ios, '12.0'
```

Comment:
@mroloux @bverbeken looks like `README.md` is out of date, it says we support minimum version iOS 9.0 and newer

Reply:
@schaloner I have updated `README.md`